### PR TITLE
Add document on Deploy Velero in Supervisor Cluster with Images from Private Registry

### PR DESCRIPTION
**Pull request by lipingxue**
Signed-off-by: Liping Xue lipingx@vmware.com

**What this PR does / why we need it**:
Add a new document about how to Deploy Velero in Supervisor Cluster with Images from Private Registry
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```NONE

```

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/827/

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/828/

**Comment by lipingxue** [comment]
@xing-yang Could you help to review this MR?

File: docs/supervisor-deploy-with-image-from-private-registry.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        *   [Images From Any Public Registry](#DeployVeleroinSupervisorClusterwithImagesfromPrivateRegistry-ImagesFromAnyPublicRegistry)
+    *   [Successful Deployment of Velero](#DeployVeleroinSupervisorClusterwithImagesfromPrivateRegistry-SuccessfulDeploymentofVelero)
+        *   [vSphere UI](#DeployVeleroinSupervisorClusterwithImagesfromPrivateRegistry-vSphereUI)
+        *   [Kubectl CLI](#DeployVeleroinSupervisorClusterwithImagesfromPrivateRegistry-KubectlCLI)
+*   [Reference](#DeployVeleroinSupervisorClusterwithImagesfromPrivateRegistry-Reference)
+
+Overview
+========
+
+In Air-gapped environment, users usually would like to deploy application from private registry which is protected by registry credential. To meet this requirement, we support to deploy Velero in Supervisor Cluster with images from private registry.
```
Comment (xing-yang) [inline docs/supervisor-deploy-with-image-from-private-registry.md:22]: we support to deploy -> we support deploying

File: docs/supervisor-deploy-with-image-from-private-registry.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+Manifest
+========
+
+*   Velero vSphere operator YAML: [https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml)
+
```
Comment (xing-yang) [inline docs/supervisor-deploy-with-image-from-private-registry.md:28]: Although the screenshots are using 1.0.0, can you add a note to refer to the latest version?

File: docs/supervisor-deploy-with-image-from-private-registry.md (hunk 1 above)
Comment (lipingxue) [reply docs/supervisor-deploy-with-image-from-private-registry.md:22]: Fixed.

File: docs/supervisor-deploy-with-image-from-private-registry.md (hunk 2 above)
Comment (lipingxue) [reply docs/supervisor-deploy-with-image-from-private-registry.md:28]: Fixed.

File: docs/supervisor-deploy-with-image-from-private-registry.md [hunk 3]
```diff
@@ -19,13 +19,15 @@
...
 
-In Air-gapped environment, users usually would like to deploy application from private registry which is protected by registry credential. To meet this requirement, we support to deploy Velero in Supervisor Cluster with images from private registry.
+In Air-gapped environment, users usually would like to deploy application from private registry which is protected by registry credential. To meet this requirement, we support deploying Velero in Supervisor Cluster with images from private registry.
 
 Manifest
 ========
 
 *   Velero vSphere operator YAML: [https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml)
 
+Note: The following screnshots show the installation using Velero vSphere Operator 1.0.0. If you want to install using the latest Velero vSphere Operator version,  please use this [YAML file](https://vmwaresaas.jfrog.io/ui/native/Velero-YAML/Velero/SupervisorService/1.4.0/velero-vsphere-1.4.0-def.yaml).
```
Comment (xing-yang) [inline docs/supervisor-deploy-with-image-from-private-registry.md:29]: Actually I think it's better to ask them to find the latest version from this page as 1.4.0 won't be the latest after we release 1.5: https://github.com/vsphere-tmm/Supervisor-Services/blob/main/README.md#velero-versions

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/831/

File: docs/supervisor-deploy-with-image-from-private-registry.md (hunk 3 above)
Comment (lipingxue) [reply docs/supervisor-deploy-with-image-from-private-registry.md]: Fixed.

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/832/

File: docs/supervisor-deploy-with-image-from-private-registry.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ Manifest
 
 *   Velero vSphere operator YAML: [https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml)
 
-Note: The following screnshots show the installation using Velero vSphere Operator 1.0.0. If you want to install using the latest Velero vSphere Operator version,  please use this [YAML file](https://vmwaresaas.jfrog.io/ui/native/Velero-YAML/Velero/SupervisorService/1.4.0/velero-vsphere-1.4.0-def.yaml).
+Note: The following screnshots show the installation using Velero vSphere Operator 1.0.0. If you want to install using the latest Velero vSphere Operator version,  please find latest version in this [page](https://github.com/vsphere-tmm/Supervisor-Services/blob/main/README.md#velero-versions).
```
Comment (xing-yang) [inline docs/supervisor-deploy-with-image-from-private-registry.md:29]: nit: "in this page" -> "on this page"
nit: double spaces before "please". 
nit: Remove "please".

Comment (lipingxue) [reply docs/supervisor-deploy-with-image-from-private-registry.md]: Fixed.

**Comment by lipingxue** [comment]
@xing-yang I have addressed your comment, please take a look. Thanks!

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/833/

File: docs/supervisor-deploy-with-image-from-private-registry.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+========
+
+In Air-gapped environment, users usually would like to deploy application from private registry which is protected by registry credential. To meet this requirement, we support deploying Velero in Supervisor Cluster with images from private registry.
+
+Manifest
+========
+
+*   Velero vSphere operator YAML: [https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml](https://vmwaresaas.jfrog.io/ui/api/v1/download?repoKey=Velero-YAML&path=Velero%252FSupervisorService%252F1.0.0%252Fvelero-supervisorservice-1.0.0.yaml)
+
+Note: The following screnshots show the installation using Velero vSphere Operator 1.0.0. If you want to install using the latest Velero vSphere Operator version, refer to this [page](https://github.com/vsphere-tmm/Supervisor-Services/blob/main/README.md#velero-versions).
```
Comment (xing-yang) [inline docs/supervisor-deploy-with-image-from-private-registry.md:29]: Typo: screnshots

Comment (lipingxue) [reply docs/supervisor-deploy-with-image-from-private-registry.md]: Good catch. Fixed and squash the commits. @xing-yang

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/834/

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/835/

**Review by xing-yang — APPROVED** [review]
lgtm

**Pull request merged**